homeassistant/components/influxdb/__init__.py

```diff
@@ -336,7 +336,7 @@ def get_influx_connection(conf, test_write=False, test_read=False):  # noqa: C90
     precision = conf.get(CONF_PRECISION)
 
     if conf[CONF_API_VERSION] == API_VERSION_2:
-        kwargs[CONF_TIMEOUT] = TIMEOUT * 1000
+        kwargs[CONF_TIMEOUT] = TIMEOUT * 1000000
```

Comment:
1 seconds = 1000 miliseconds; You've just concerted into microseconds, that doesn't sound right?

Reply:
There was a `*1000` here before and I was told it was expecting `TIMEOUT` in milliseconds here while the `v1` section expects seconds. I was trying it to make it use seconds everwhere in `v1` and here in `v2`. If it was `*1000` before, that would mean changing it to `*1000000`. Sorry if I misunderstood.

Reply:
Before this PR `kwargs[CONF_TIMEOUT]` was set to 5000 - apparently, meaning 5000ms or 5s
Now you set the same variable to 10000000 - i.e., increase it by a factor of 2000. It doesn't seem right.

Reply:
Even if you changed `InfluxDBClientV2` to expect seconds instead of milliseconds (which you didn't), you would need to send 10 there instead of 10000000. :)

Reply:
The `* 1000` was already converting seconds into milliseconds. That conversion was already done, you don't need to change this code at all.

Reply:
> you don't need to change this code at all.

Ok, changed it back then.